models/DataObject/ClassDefinition/Data/Block.php

```diff
@@ -508,12 +496,8 @@ public function getFromWebserviceImport($value, $relatedObject = null, $params =
             foreach ($value as $blockElementsData) {
                 $resultElement = [];
 
-                /**
-                 * @var  $blockElement DataObject\Data\BlockElement
-                 */
+                /** @var DataObject\Data\BlockElement $blockElement */
```

Comment:
$blockElementDataRaw

Reply:
$blockElementDataRaw should be a array. $blockElement is a few lines below, we can remove this.